SeQuant/domain/mbpt/vac_av.ipp

```diff
@@ -77,7 +77,7 @@ ExprPtr vac_av(
     expr = simplify(expr);
 
     // compute VEV
-    return vac_av(expr, connections, /* use_topology = */ true);
+    return simplify(vac_av(expr, connections, /* use_topology = */ true));
```

Comment:
when debugging often these chained function calls make life more difficult, I prefer more verbose
```
auto vev = vac_av();
return simplify(vev); // simplify since vac_av does not
```